src/openforms/forms/api/serializers/form.py

```diff
@@ -55,6 +55,45 @@ class FormLiteralsSerializer(serializers.Serializer):
     confirm_text = ButtonTextSerializer(raw_field="confirm_text", required=False)
 
 
+class FormAuthenticationBackendSerializer(serializers.ModelSerializer):
+    options = serializers.DictField(label=_("authentication backend options"))
```

Comment:
I'd go with a `serializers.JSONField` so that we have full freedom about the shape of plugin-specific options.